docs/source/concepts/cl_columns.rst

```diff
@@ -21,37 +28,99 @@ The Cleanlab columns are listed below in the table. Detailed explanations of eac
 
    * - Cleanlab Column Name
      - Value Type
-   * - :ref:`label_issue_score <label_issue_score>`
-     - Float
    * - :ref:`is_label_issue <is_label_issue>`
      - Boolean
+   * - :ref:`label_issue_score <label_issue_score>`
+     - Float
    * - :ref:`suggested_label <suggested_label>`
      - String
    * - :ref:`is_outlier <is_outlier>`
      - Boolean
+   * - :ref:`outlier_score <outlier_score>`
+     - Float
+   * - :ref:`is_ambiguous <is_ambiguous>`
+     - Boolean
+   * - :ref:`ambiguous_score <ambiguous_score>`
+     - Float
+   * - :ref:`is_high_confidence_given_label <is_high_confidence_given_label>`
+     - Boolean
+   * - :ref:`is_near_duplicate <is_near_duplicate>`
+     - Boolean
+   * - :ref:`near_duplicate_score <near_duplicate_score>`
+     - Float
+   * - :ref:`near_duplicate_id <near_duplicate_id>`
+     - Integer
 
 Label Issues
 ============
 
-.. _label_issue_score:
-``label_issue_score``
--------------
-Contains a score bounded between 0 and 1. The score is calculated using confident learning. The lower the score of a sample, the more likely it has a label issue.
-
 .. _is_label_issue:
 ``is_label_issue``
 -----
 Contains a boolean value, with ``True`` indicating that the sample is likely to have a label issue. The value is obtained by thresholding the ``label_issue_score`` score with confident learning.
 
+.. _label_issue_score:
+``label_issue_score``
+-------------
+Contains a score bounded between 0 and 1. The score is calculated using confident learning. The lower the score of a sample, the more likely it has a label issue.
+
 .. _suggested_label:
 ``suggested_label``
 ---------------
-Contains the suggested label for the sample. If the sample is not a label issue (``is_label_issue == False``), the suggested label will be empty. For samples with label issues, the suggested label is computed by Cleanlab studio.
+Contains the suggested label for the sample. If the sample is not a label issue (``is_label_issue`` marked as ``False``), the suggested label will be empty. For samples with label issues, the suggested label is computed by Cleanlab studio.
 
 Outliers
 ========
 
 .. _is_outlier:
 ``is_outlier``
 -------
-Contains a boolean value, with ``True`` indicating that the sample is likely to be an outlier.
\ No newline at end of file
+Contains a boolean value, with ``True`` indicating that the sample is likely to be an outlier.
+
+Note: for projects on multi-label tabular datasets, outliers are currently not computed, and ``is_outlier`` will always be False. 
```

Comment:
This note renders as gigantic text

<img width="841" alt="Screenshot 2023-06-27 at 3 21 29 PM" src="https://github.com/cleanlab/cleanlab-studio/assets/3526486/ea9fd175-8c94-45d4-9ad3-daed6837e30f">